pkg/volume/azure_dd/azure_dd_max_disk_count.go

```diff
@@ -0,0 +1,227 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azure_dd
+
+var maxDataDiskCountMap = map[string]int64{
```

Comment:
Do we have any scripts/commands to generate the list? If so, we should include it in the comments, so that we can easily update the list

Reply:
> > BTW, there is a side effect for this PR, with this PR, new released VM type would have default max data disk count(16), need to update azure_dd_max_disk_count.go file periodically. @khenidak
> 
> I think this is ok. When new VM type released, we can patch Kubernetes with latest count.

that's for master branch, we can always get the latest count, while for old release, the GetVolumeLimit function would not work for new release VM type.

Reply:
added the doc link about how to generate vm size list.

Reply:
The ways listed on the issue are somewhat too complicated, an easier way is:
```
az vm list-sizes -l <location> -o json | jq '.[] | {name, maxDataDiskCount}'
```

Besides this, we need another wrapper for merging vmsizes from different locations.

Reply:
yes, it looks like we dig a hole that is error-prone... need to merge vmsizes for all azure regions periodically, and when new vm size comes out, need to patch for old release, otherwise `GetVolumeLimit` functionality is wrong on new vm type...
cc @khenidak

Reply:
Add link `https://raw.githubusercontent.com/andyzhangx/demo/master/linux/azuredisk/get-vm-size-list.sh` to this comments?

Reply:
added another link, it's fixed.